dove/src/cmd/ct.rs

```diff
@@ -617,7 +622,9 @@ fn store_transaction(ctx: &Context, name: &str, tx: Transaction) -> Result<(), E
     }
 
     let mut tx_file = tx_dir.join(name);
-    tx_file.set_extension("mvt");
+    if !name.ends_with(".mvt") {
```

Comment:
Maybe we should do `to_lowercase` here?
_(and here only)_

Reply:
Thanks. fixed it.